includes/country-functions.php

```diff
@@ -239,7 +239,7 @@ function give_get_country_list() {
 		'NR' => esc_html__( 'Nauru', 'give' ),
 		'NP' => esc_html__( 'Nepal', 'give' ),
 		'NL' => esc_html__( 'Netherlands', 'give' ),
-		'AN' => esc_html__( 'Netherlands Antilles', 'give' ),
+		'CW' => esc_html__( 'Curaçao', 'give' ),
```

Comment:
The tricky part about this is the `Netherlands Antilles` country code can be assigned/stored in the give settings for people and removing it could break something for them when updating.  I would suggest keeping the old one and adding this new one so people can choose to update themselves